server/src/main/java/org/apache/gravitino/server/web/rest/FilesetOperations.java

```diff
@@ -268,7 +269,7 @@ public Response getFileLocation(
         catalog,
         schema,
         fileset,
-        subPath);
+        RESTUtils.decodeString(subPath));
```

Comment:
Make the server-side encode/decode unit test more complete. In addition, the current integration test also covers the normal situation of encode/decode, otherwise the `actualFileLocation` returned by the server cannot match the `Location` expected by the client.